worldedit-core/src/main/java/com/sk89q/worldedit/internal/util/ErrorReporting.java

```diff
@@ -31,11 +31,13 @@
 public class ErrorReporting {
     public static void trigger(Actor actor, Throwable error) {
         actor.printError(TranslatableComponent.of("worldedit.command.error.report"));
-        actor.print(
-            TextComponent.builder(error.getClass().getName() + ": " + error.getMessage())
-                .hoverEvent(HoverEvent.showText(TextComponent.of(Throwables.getStackTraceAsString(error))))
-                .build()
-        );
+        if (actor.hasPermission("worldedit.error.detailed")) {
+            actor.print(
+                TextComponent.builder(error.getClass().getName() + ": " + error.getMessage())
+                    .hoverEvent(HoverEvent.showText(TextComponent.of(Throwables.getStackTraceAsString(error))))
```

Comment:
Maybe we should only put the hover event behind the permission, as it was before. @wizjany & @me4502 thoughts?

Reply:
yea i think the message itself is still useful for the user tbh

Reply:
Yeah IMO just doing prior behaviour without the perm node makes sense to me.